modules/serverless-kafka-broker-with-kafka-topic.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-kafka-developer.adoc
+// * serverless/develop/serverless-using-brokers.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-broker-with-kafka-topic_{context}"]
+= Creating a Kafka broker that uses an existing Kafka topic
+
+If you want to use a Kafka broker without creating a new topic, you can use an existing Kafka topic instead. To do this, you must create a Kafka `Broker` object that uses the `kafka.eventing.knative.dev/external.topic` annotation.
```

Comment:
> If you want to use a Kafka broker without creating a new topic

The broker does create the topic for the messages, it's not the user that does create the topic here. That might be a bit confusing to read, I guess.

How about:

If you want to use a Kafka broker without having it creating its own internal topic, you can use an existing Kafka topic instead.

(or externally managed topic, instead of "existing kafka topic")?